Tests/SwiftParserTest/IncrementalParsingTests.swift

```diff
@@ -16,31 +16,85 @@ import SwiftParser
 
 public class IncrementalParsingTests: XCTestCase {
 
+  public func testDiffOfTwoStringsSimple() {
+    let s1 = "struct A { func f() {"
+    let s2 = "struct AA { func f() {"
+
+    let diffs = getDiffOfTwoString(s1, s2)
+    XCTAssert(diffs.count == 1)
+
+    let firstDiff = diffs.first!
+    XCTAssert(firstDiff == SourceEdit(offset: 8, length: 0, replacementLength: 1))
+  }
+
+  public func testDiffOfTwoSameStrings() {
+    let s1 = "0123456"
+    let s2 = "0123456"
+
+    let diffs = getDiffOfTwoString(s1, s2)
+    XCTAssert(diffs.count == 0)
+  }
+
+  public func testDiffOfTwoStrings() {
+    let s1 = "0123456"
+    let s2 = "x12456yz"
+
+    // should return [(0, 1, 0), (0, 0, 1), (3, 1, 0), (6, 0, 1), (7, 0, 1)]
```

Comment:
Do you think this comment provides much value? It’s exactly what you are specifying below in `expectedDiffs` and in my experience these comments just get out-of-sync. 😆

Reply:
That comment was used to remind myself what `expectedDiffs` should be when writing test case lol. Already dropped it.😄